django/forms/utils.py

```diff
@@ -49,12 +51,17 @@ def get_context(self):
         )
 
     def render(self, template_name=None, context=None, renderer=None):
-        return mark_safe(
-            (renderer or self.renderer).render(
-                template_name or self.template_name,
-                context or self.get_context(),
+        renderer = renderer or self.renderer
+        template = template_name or self.template_name
+        if template == "django/forms/default.html":
```

Comment:
It will also catch folk who have overridden the default.html template.

Previous discussion suggested  to add a release not for this.  

https://github.com/django/django/pull/15241#pullrequestreview-934786953

Maybe the release note can be extended to the `extends` scenario. Are there any other edge cases here? 🤔 (personally I can't think of any fwiw)